Dockerfile

```diff
@@ -13,23 +13,44 @@ WORKDIR /usr/src/app
 # A wildcard is used to ensure both package.json AND package-lock.json are copied
 # where available (npm@5+)
 COPY package*.json ./
+# Expose port 4000 inside the container to the outside world
+# so that http://localhost:3000 routes the network traffic to
+# the container
```

Comment:
Can we remove this or make the comment correct. Port 4000 vs 3000

Reply:
oops